spec/lib/farscape/plugins_spec.rb

```diff
@@ -223,6 +224,64 @@ def call(env)
 
     end
 
+    context 'extensions' do
+      before(:each) { Farscape.clear }
+      after(:all) { Farscape.clear }
+        
+      let(:entry_point) { "http://localhost:#{RAILS_PORT}"}
+      let(:can_do_hash) { {conditions: 'can_do_anything'} }
+      
+      it 'allows extensions' do
+        module Peacekeeper
+          def pacify!
+            raise 'none shall pass' if enabled_plugins.include?(:Peacekeeper)
```

Comment:
I don't think we talked about this.  It was just me putting something in I could mess with.  It also happens to be something that can never evaluate to false since the extension wouldn't be used if it wasn't disabled.  It would probably be more interesting if it were checking whether some other plugin was enabled.